requirements.txt

```diff
@@ -9,3 +9,5 @@ transformers>=4.6.0
 sentencepiece>=0.1.95
 nmslib>=2.1.1
 onnxruntime>=1.8.1
+spacy==3.0.6
```

Comment:
@MXueguang @ronakice @yuki617 - what do you think? should it be pegged to a version?

`>=` instead? Or just unspecified?

FWIW, my system:

```
lightgbm           3.3.2
spacy              3.2.1
```

Reply:
ah yes, I think `>=` is a better idea
if using `>=`, should we use newer version instead of my old ones

Reply:
@ronakice  does it affect the 2.1.6 restriction for d2q?

Reply:
I see. yeah then >= 3.2.1 would be good

Reply:
So @stephaniewhoo let's do

```
lightgbm>=3.3.2
spacy>=3.2.1
```

Reply:
yeah, for v1 d2q with the old stuff we can just leave it and mention in the repository clearly that we used that.

Reply:
Yea, I believe spacy>=3.2.1 should be good.